.gitignore

```diff
@@ -114,3 +114,4 @@ ENV/
 *.code-workspace
 
 .DS_Store
+setup_env.sh
```

Comment:
Where does `setup_env.sh` come from?

Reply:
That's a file that we kept locally to configure environment variables and start the Docker container so that we can get a working test environment quickly.

Reply:
Given this is a local workflow, we should exclude in the codebase/repo.

Reply:
Removed by @ximenesuk